ui/apps/dashboard/src/app/intent/create-webhook/page.tsx

```diff
@@ -22,26 +24,36 @@ const CreateWebhook = graphql(`
   }
 `);
 
+function getNameFromDomain(domain: string | null) {
+  if (!domain) return '';
+  const removeTLD = domain.replace(/\.com|\.io|\.org|\.net|\.co\..{2}|\.dev|\.app|\.ai|\.xyz$/, '');
```

Comment:
This won't cover it all, but that's ok, since the user can still edit the prefix, we just make a best attempt.